use-cases/product_ratings_with_pipelines/pipelines_product_ratings.ipynb

```diff
@@ -0,0 +1,1310 @@
+{
```

Comment:
<p>Debugger does not support Blazing Text algo. We need to remove the Debugger hook configs.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2884/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>